core/src/config.jl

```diff
@@ -134,6 +128,18 @@ end
     logging::Logging = Logging()
 end
 
+function Configurations.from_dict(::Type{Logging}, ::Type{LogLevel}, level::AbstractString)
+    level == "debug" && return LogLevel(-1000)
+    level == "info" && return LogLevel(0)
+    level == "warn" && return LogLevel(1000)
+    level == "error" && return LogLevel(2000)
+    throw(
+        ArgumentError(
+            "verbosity $level not supported, choose one of: debug info warn error.",
```

Comment:
For consistency maybe also use this, like above:
`$(join(instances(Compression), " "))`

Reply:
There is no such thing for LogLevel I believe